ewah/dag_factories/dbt_dag_factory.py

```diff
@@ -23,8 +25,9 @@ def dbt_dags_factory(
     schema_name="analytics",  # see https://docs.getdbt.com/dbt-cli/configure-your-profile/#understanding-target-schemas
     keepalives_idle=0,  # see https://docs.getdbt.com/reference/warehouse-profiles/postgres-profile/
     dag_base_name="T_dbt_run",
-    schedule_interval=timedelta(hours=1),
+    schedule_interval: Union[str, timedelta] = timedelta(hours=1),
     start_date=datetime(2019, 1, 1),
+    end_date=None,
```

Comment:
Request (blocking): Can you either

1. Not have `end_date` as kwarg and define it as `None` for cron-style schedule intervals, or
2. Respect `end_date` in the normal timedelta case

Reply:
second case happens in the else statement in line 64, no?